pkg/cmd/service/service_test.go

```diff
@@ -671,8 +676,8 @@ func TestGRPCServingTLS(t *testing.T) {
 		ensureServiceUp(t, creds)
 
 		cancel()
-		require.NoError(t, g.Wait())
 		require.NoError(t, service.Close(ctx))
+		require.NoError(t, g.Wait())
```

Comment:
👍🏽 good catch.

Reply:
Come to think of it, why are we `canceling` if we are actually gracefully closing right after? Seems a bit of a mix up there.

Reply:
`canlceling` is the thing that is triggering the graceful closure. Server.Close() closes database and auth connections only. It's a bit ... odd maybe?